.github/workflows/reusable-agent-build-container-images.yml

```diff
@@ -579,17 +579,6 @@ jobs:
           ./scripts/cicd/scalyr-query.sh '$serverHost="'${SCALYR_AGENT_POD_NAME}'" $logfile="/var/log/scalyr-agent-2/kubernetes_events.log" $parser="k8sEvents" k8s-kind="CronJob" involvedObjectKind="CronJob" involvedObjectName="hello" watchEventType="ADDED" reason="SawCompletedJob"'
           
           ./scripts/cicd/scalyr-query.sh '$serverHost="'${SCALYR_AGENT_POD_NAME}'" "Hello from the Kubernetes cluster"'
-
```

Comment:
I believe that we should still have an integration test for short lived pods and including those pods based on SCALYR_K8S_INCLUDE_ALL_CONTAINERS.

Reply:
Ok, ready for another look